### PR TITLE
Fix Debug LDAP Informations on User

### PR DESCRIPTION
**Pull request by tsmr**
- [X ] I have read the CONTRIBUTING document.
- [X ] I have performed a self-review of my code.

## Description

When you activate the debug mode on a User, LDAP informations are displayed. But in some cases, the memberof attribute don't be displayed because the LdapSearch don't request operational attributes

https://docs.ldap.com/ldap-sdk/docs/tool-usages/ldapsearch.html

## Screenshots (if appropriate):

image

### CODE REVIEW

File: src/User.php [hunk 1]
```diff
@@ -5965,7 +5965,7 @@ private function showLdapInformation(): void
                 $info = AuthLDAP::getUserByDn(
                     $ds,
                     $this->fields['user_dn'],
-                    ['*', 'createTimeStamp', 'modifyTimestamp']
+                    ['*', '+', 'createTimeStamp', 'modifyTimestamp']
```
Comment (cedric-anne) [inline src/User.php:5968]: Could you explain and even give a link to a documentation that explains what is `+` means ?

Comment (cedric-anne) [reply src/User.php:5968]: My bad, you already did.

**Comment by trasher** [comment]
I do not have any LDAP or similar installed, so I cannot really tests; but that looks OK.

This should probably target 10.0/bf branch.

File: src/User.php (hunk 1 above)
Comment (cedric-anne) [inline src/User.php:5968]: 1. We could add some documentation here
2. I just test with openldap and `createTimeStamp`/`modifyTimestamp` are already returned by the `+`  as they are operational attributes. I guess it is the same for all LDAPs servers.

```suggestion
                    [
                        // see https://docs.ldap.com/ldap-sdk/docs/tool-usages/ldapsearch.html
                        '*', // all user attributes
                        '+', // all operational attributes
                    ]
```

**Review by trasher — APPROVED** [review]

**Review by cedric-anne — APPROVED** [review]

**Pull request merged**